pkg/ipam/pool.go

```diff
@@ -222,89 +193,6 @@ func (p *podCIDRPool) releaseExcessCIDRsMultiPool(neededIPs int) {
 	p.ipAllocators = retainedAllocators
 }
 
-// releaseExcessCIDRsLocked implements the logic for clusterpool-v2-beta
-func (p *podCIDRPool) releaseExcessCIDRsLocked(totalFree, releaseThreshold int) {
-	// Iterate over pod CIDRs in reverse order, so we prioritize releasing
-	// later pod CIDRs.
-	for i := len(p.ipAllocators) - 1; i >= 0; i-- {
-		ipAllocator := p.ipAllocators[i]
-		cidrNet := ipAllocator.CIDR()
-		cidrStr := cidrNet.String()
-		if _, released := p.released[cidrStr]; released || ipAllocator.Used() > 0 {
-			// CIDR is either in use or already released
-			continue
-		}
-
-		if _, removed := p.removed[cidrStr]; removed {
-			// If the pod CIDR has been removed, then release it
-			p.released[cidrStr] = struct{}{}
-			delete(p.removed, cidrStr)
-			log.WithField(logfields.CIDR, cidrStr).Debug("releasing removed pod CIDR")
-		} else if free := ipAllocator.Free(); totalFree-free >= releaseThreshold {
-			// Otherwise, if the pod CIDR is not used and releasing it would
-			// not take us below the release threshold, then release it and
-			// mark it as released.
-			p.released[cidrStr] = struct{}{}
-			totalFree -= free
-			log.WithField(logfields.CIDR, cidrStr).Debug("releasing pod CIDR")
-		}
-	}
-}
-
-func (p *podCIDRPool) clusterPoolV2Beta1Status(allocationThreshold, releaseThreshold int) types.PodCIDRMap {
-	p.mutex.Lock()
-	defer p.mutex.Unlock()
-
-	if allocationThreshold <= 0 {
-		allocationThreshold = defaults.IPAMPodCIDRAllocationThreshold
-	}
-
-	if releaseThreshold <= 0 {
-		releaseThreshold = defaults.IPAMPodCIDRReleaseThreshold
```

Comment:
`defaults.IPAMPodCIDRAllocationThreshold` and `defaults.IPAMPodCIDRReleaseThreshold` now seem unused, probably they can be deleted as well?

Reply:
Oh yeah, good catch! Those need to be removed as well!